### PR TITLE
fix: move documentation of Portal API to RF docs

### PR DESCRIPTION
**Pull request by omarspx**
## Description of the changes

Removed all references to Portal API documentation. Cannot merge until Portal API is available on Request Finance.

### CODE REVIEW

**Comment by omarspx** [comment]
Renamed folders number with the following:

image

Protocol is now 1 instead of 7

File: packages/docs/src/components/integration-options/index.js [hunk 1]
```diff
@@ -59,7 +59,17 @@ const options = [
...
     ),
-  }
+  },
+  {
+    title: 'Option C: Portal API',
+    imageUrl: 'img/ReQ-01.png',
+    description: <>Fully managed solution, over REST API using Request Finance</>,
+    details: (
+      <>
+        <p>Reach out to us if you want to integrate Request Finance APIs.</p>
```
Comment (omarspx) [inline packages/docs/src/components/integration-options/index.js:69]: Readded Portal API as an option for now.

**Review by bertux — APPROVED** [review]

**Review by alexandre-abrioux — COMMENTED** [review]
I will review it again once the portal API doc is transferred to the `request.finance` domain. Let's wait for the migration before merging this 👍

**Comment by bertux** [comment]
> I will review it again once the portal API doc is transferred to the `request.finance` domain. Let's wait for the migration before merging this +1

Now that https://docs.request.finance is deployed could you take time to review it again ?

File: packages/docs/src/components/integration-options/index.js (hunk 1 above)
Comment (alexandre-abrioux) [reply packages/docs/src/components/integration-options/index.js:69]: You can link to https://docs.request.finance if you want.
I will also link docs.request.network in docs.request.finance

**Review by alexandre-abrioux — APPROVED** [review]

**Review by alexandre-abrioux — APPROVED** [review]

**Review by MantisClone — APPROVED** [review]

**Pull request closed without merging**